pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -228,31 +228,30 @@ func (pl *DefaultPreemption) SelectVictimsOnNode(
 // considered for preemption.
 // We look at the node that is nominated for this pod and as long as there are
 // terminating pods on the node, we don't consider this for preempting more pods.
-func (pl *DefaultPreemption) PodEligibleToPreemptOthers(pod *v1.Pod, nominatedNodeStatus *framework.Status) bool {
```

Comment:
explain in the comment what are the return values.

Reply:
done